src/channels.c

```diff
@@ -355,7 +355,7 @@ const idclass_t channel_class = {
     {
       .type     = PT_BOOL,
       .id       = "autoname",
-      .name     = N_("Auto Name"),
+      .name     = N_("Auto name from network"),
```

Comment:
Just wondering if we should be more consistent on "auto" versus "automatic/ally"... i.e. should this be "Automatically name from network" to balance "Automatically map to EPG source"